TypeCobol/Compiler/File/SourceFileProvider.cs

```diff
@@ -30,6 +30,14 @@ public SourceFileProvider()
             CobolLibraries = new List<ICobolLibrary>();
         }
         
+        /// <summary>
+        /// Clear all Cobol Libraries
+        /// </summary>
+        public void Clear()
```

Comment:
Naming: `RemoveAllLibraries` because `Clear` is intended for a collection like class. `SourceFileProvider` is not a collection but a library manager

Reply:
Done [4102038](https://github.com/TypeCobolTeam/TypeCobol/pull/2145/commits/4102038a30a2c65ba3e94f71551a3ad68c37f746)